### DISCUSSION TITLE
Appreciation thread for how far Waveform has come.

### DISCUSSION THREAD

**Discussion by mikobuntu** [Tracktion/tracktion_engine#191] [General]
Hi guys, I always feel like I´m complaining about this being broken or I need that function etc ( I guess that´s the life of a Beta tester ),but I would like to say a few things that i have noticed about the general direction that Waveform has taken since I have been onboard.

I found Waveform around version 8 ( well it was obviously called Tracktion then )  I think and I loved the look of it, but knew some stuff needed fixing.... A lot of those things have been fixed and then some !!

I use a lot of DAWs ( mostly on Linux ), but Waveform has become my go to when I want to put down an idea and turn it into a full song easily.

The Developers are amazing guys, they actually listen to their users and work fast ( trust me, I have asked for FRś from a lot of other companies and they basically just ignore you.... the only other Devs in the ballpark are Reaperś , who are very similar to TSC ).

Bug fixing is done fast and feedback with Devs is taken seriously !!!

There are obviously some issues that I think if addressed, would push Waveform out there even further, but I´ll keep that for another topic.

Anyway would be good to hear my fellow Beta tester friends opinions on this.

And once again thanks guys for an amazing DAW !!!

**Comment by drowaudio** [comment]
Thanks for the kind words @mikobuntu, it's easy to forget the huge changes since W8 (I think that was the first version of "Waveform") when we see it evolve daily but we really have worked hard to gradually add missing chunks of features over the years so it's nice to hear this hasn't gone unnoticed.

I am always keen to hear what bigger features (or smaller ones) people would like to see and how they might fit in to Waveform. Feel free to start a Discussion on that and we'll check back when we have bandwidth.

I'd also like to take this space to say thank you to all of the incredible beta team. There is absolutely no way we would have been able to come this far with you. So thank you so much for giving up your valuable time and often interrupting your creative flow to report issues.

I know the development process can be a bit opaque to the testing team sometimes so apologies for that, we try to be as transparent as possible but there's a lot of moving parts behind the scenes we need to balance and some Issues are more complicated to dig in to thank they might seem on the surface. The level of bug reporting and detailed feedback on here is top notch so 🙏.

Cheers!
